python_websocket_server/tests/test_app.py

```diff
@@ -0,0 +1,37 @@
+import ast
+
+import pytest
+from pyhocon import ConfigFactory
+from sanic.websocket import WebSocketProtocol
+
+from deepspeech_server.app import app
+
+# Load app configs and initialize DeepSpeech model
+conf = ConfigFactory.parse_file("application.conf")
+
```

Comment:
The formatting might look a bit strange if you're not used to seeing two blank lines, but I assure you it's "conventionally" formatted. I'm using the popular [black](https://github.com/psf/black) Python formatter.